literate-unitb-config/Data/Bidirectional.hs

```diff
@@ -21,6 +23,7 @@ instance (Applicative f) => Applicative (BiParser f a b) where
 
 -- instance Profunctor (BiParser a) where
```

Comment:
Let's remove this commented code. If we need a `Profunctor` instance, we'll just add it from scratch